diff -Naur /ubuntu/test/pulseaudio-4.0/src/pulsecore/mutex-posix.c /ubuntu/pulse/pulseaudio-4.0/src/pulsecore/mutex-posix.c
--- /ubuntu/test/pulseaudio-4.0/src/pulsecore/mutex-posix.c	2012-07-03 12:47:50.000000000 +0000
+++ /ubuntu/pulse/pulseaudio-4.0/src/pulsecore/mutex-posix.c	2013-09-14 06:32:12.000000000 +0000
@@ -49,19 +49,8 @@
     if (recursive)
         pa_assert_se(pthread_mutexattr_settype(&attr, PTHREAD_MUTEX_RECURSIVE) == 0);
 
-#ifdef HAVE_PTHREAD_PRIO_INHERIT
-    if (inherit_priority) {
-        r = pthread_mutexattr_setprotocol(&attr, PTHREAD_PRIO_INHERIT);
-        pa_assert(r == 0 || r == ENOTSUP);
-    }
-#endif
-
     m = pa_xnew(pa_mutex, 1);
 
-#ifndef HAVE_PTHREAD_PRIO_INHERIT
-    pa_assert_se(pthread_mutex_init(&m->mutex, &attr) == 0);
-
-#else
     if ((r = pthread_mutex_init(&m->mutex, &attr))) {
 
         /* If this failed, then this was probably due to non-available
@@ -72,7 +61,7 @@
         pa_assert_se(pthread_mutexattr_setprotocol(&attr, PTHREAD_PRIO_NONE) == 0);
         pa_assert_se(pthread_mutex_init(&m->mutex, &attr) == 0);
     }
-#endif
+
 
     return m;
 }
